server/common.js

```diff
@@ -17,7 +17,7 @@ const userTypes = Object.freeze({
 exports.userTypes = userTypes;
 
 const questionTypes = Object.freeze({
-    REGULAR         : 0,
-    MULTIPLECHOICE  : 1
+    REGULAR         : {name: 'Regular Question', value: "re", template: 'regex-answer'},
+    MULTIPLECHOICE  : {name: 'Multiple Choice', value: "mc", template: 'mc-answer'}
```

Comment:
Line 20-21 mixed quotations, stick to single quotes?

Reply:
yes please, let's use single quotes everywhere in `.js` files